tests/test_parsers.py

```diff
@@ -223,3 +224,25 @@ def test_read_sssom_table(self):
                         self.assertEqual(imported_df.iloc[idx][k], v)
                     else:
                         self.assertEqual(imported_df.iloc[idx][k], v)
+
+    def test_parse_obographs_merged(self):
+        """Test parsing OBO Graph JSON using custom prefix_map."""
+        hp_json = f"{test_data_dir}/hp-subset.json"
+        hp_meta = f"{test_data_dir}/hp-subset-metadata.yml"
+        outfile = f"{test_out_dir}/hp-subset-parse.tsv"
+
+        with open(hp_meta, "r") as f:
+            data = yaml.safe_load(f)
+            custom_curie_map = data["curie_map"]
+
+        with open(outfile, "w") as f:
+            parse_file(
+                input_path=hp_json,
+                prefix_map_mode="merged",
+                clean_prefixes=True,
+                input_format="obographs-json",
+                metadata_path=hp_meta,
+                output=f,
+            )
+        msdf = parse_sssom_table(outfile)
+        self.assertDictContainsSubset(custom_curie_map, msdf.prefix_map)
```

Comment:
I think that assert method is deprecated in python 3.11.. again.

https://stackoverflow.com/questions/20050913/python-unittests-assertdictcontainssubset-recommended-alternative

```suggestion
        self.assertTrue(custom_curie_map.items() <= msdf.prefix_map.items())
```

Reply:
(untested code)